aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/service/AWSS3StorageService.java

```diff
@@ -249,21 +247,18 @@ public void resumeTransfer(@NonNull TransferObserver transfer) {
      * @param transfer A file transfer to cancel
      */
     public void cancelTransfer(@NonNull TransferObserver transfer) {
-        startServiceIfNotAlreadyStarted();
+        startTransferService();
         transferUtility.cancel(transfer.getId());
     }
 
-    private void startServiceIfNotAlreadyStarted() {
-        if (!transferUtilityServiceStarted) {
-            // TODO: When a reset method is defined, stop service.
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-                Intent serviceIntent = new Intent(context, TransferService.class);
-                serviceIntent.putExtra(TransferService.INTENT_KEY_NOTIFICATION, createDefaultNotification());
-                context.startForegroundService(serviceIntent);
-            } else {
-                context.startService(new Intent(context, TransferService.class));
-            }
-            transferUtilityServiceStarted = true;
+    private void startTransferService() {
+        // TODO: When a reset method is defined, stop service.
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            Intent serviceIntent = new Intent(context, TransferService.class);
+            serviceIntent.putExtra(TransferService.INTENT_KEY_NOTIFICATION, createDefaultNotification());
```

Comment:
This notification says "Amplify transfer service is running". This will be out of context of the App that developed using Amplify. For App end users they won't have an idea about "Amplify".

Do we need this notification here? If this is required by Android specs, can this message be customized to fit into the App?

Reply:
"Amplify transfer service is running" is a string resource that can be overridden, but further discussion is required on how we want to enable this. Changes could include programatic notification customizations, or adding string resources to documentation (which would warrant commitment on keeping the string resource ids stable). The notification is shown to keep the transfer alive if the app is moved to the background.

Reply:
Thanks for the explanation, programatic notification customizations sounds more feasible for amplify-flutter to consume amplify-android. Flutter developers don't have direct access to override Android resources...